src/toolboxes/paintbrush/Submenu.tsx

```diff
@@ -52,11 +55,13 @@ const Submenu = (props: Props): ReactElement => {
           showEndValues={false}
         />
       </div>
-      <BaseIconButton
-        tooltip={tooltips.fillbrush}
-        onClick={fillBrush}
-        fill={false}
-      />
+      <div className={classes.fillDiv}>
```

Comment:
Does this need an extra div? Can't you just assign the classname to the div the slider is in and move the button there or the popeover itself? Don't forget we should be keeping the DOM tree as simple as possible?